versions/v6/functions.php

```diff
@@ -179,11 +179,11 @@ function display_photo_essay_item( $orientation, $item_id, $image_url, $title, $
 		case 'portrait':
 	?>
 		<div class="row">
-			<div class="img-col col-lg-7 offset-lg-0 col-md-10 offset-md-1 <?php if ( $alternate ) { ?>col-lg-push-5<?php } ?>">
+			<div class="img-col col-lg-7 offset-lg-0 col-md-10 offset-md-1 <?php if ( $alternate ) { ?>push-lg-5<?php } ?>">
```

Comment:
Not sure if we have a set preference on this, but I think ordering col classes from smallest to largest helps organize things a bit:
```suggestion
			<div class="img-col col-md-10 offset-md-1 col-lg-7 offset-lg-0 <?php if ( $alternate ) { ?>push-lg-5<?php } ?>">
```